_episodes/44.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: episode
+title: "44: Seeing clearly with Metabase"
+date: 2023-01-26
+tags: trino metabase visualization client clojure bi reporting
+youtube_id: ""
+wistia_id: ""
+sections:
+   - title: "Release update"
+     desc: "405 and 406"
+     time: 
+---
+
+## Hosts
+
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guests
+
+* [Luis Paolini](https://www.linkedin.com/in/luispaolini/), Product Advocate at
```

Comment:
@paoliniluis .. is this ok? should I add some other links like twitter or so?

Reply:
I would change it to Success Engineer